usr/src/uts/common/fs/zfs/spa.c

```diff
@@ -6985,6 +6987,14 @@ spa_event_create(spa_t *spa, vdev_t *vd, const char *name)
 		}
 	}
 
+	if (hist_nvl) {
```

Comment:
I would prefer a more explicit '!= NULL' comparison, but perhaps this matches the local style expectations.